package.json

```diff
@@ -78,7 +78,6 @@
         "babel-plugin-transform-require-ignore": "^0.1.1",
         "box-node-sdk": "^1.22.0",
         "circular-dependency-plugin": "^5.2.0",
-        "codeceptjs-webdriverio": "^1.1.0",
```

Comment:
There's a bunch more stuff in here that can be removed. Take a look at all the helper libraries, like `pug` and whatnot.